convert_hf_to_gguf.py

```diff
@@ -5262,6 +5262,108 @@ def prepare_tensors(self):
                 raise ValueError(f"Unprocessed experts: {experts}")
 
 
+@ModelBase.register("Dots1ForCausalLM")
+class DotsModel(TextModel):
```

Comment:
There is a more simple way for the conversion code: https://github.com/ngxson/llama.cpp/blob/b469d9b86e148c4d7538ad27f817cf83bc2fb339/convert_hf_to_gguf.py#L3070-L3087